controller/uninstall_controller.go

```diff
@@ -410,6 +410,14 @@ func (c *UninstallController) deleteCRDs() (bool, error) {
 		return true, nil
 	}
 
+	// Delete the BackupTarget CRs
+	if backupTargets, err := c.ds.ListBackupTargets(); err != nil {
+		return true, err
+	} else if len(backupTargets) > 0 {
+		c.logger.Infof("Found %d backuptargets remaining", len(backupTargets))
+		return true, c.deleteBackupTargets(backupTargets)
+	}
```

Comment:
Leave a note here
```
Derek Su  3:22 PM
Why didn’t we delete the backupTarget CRs before?

JenTing Hsiao  3:24 PM
Because the BackupTarget CR without finalizer
So, it fine when we delete CRD, the CR will be deleted as well.
However, once the webhook introduce, if we keep the same that the BackupTarget CR be deleted when we delete CRD.
After the uninstallation job finished, we can't delete BackupTarget CR success anymore because the longhorn-webhook server gone.
```